pkg/provider/aws/plugin.go

```diff
@@ -33,6 +33,11 @@ func (a *AWS) Translate(result *core.ConstructGraph, dag *core.ResourceGraph) (L
 			if err != nil {
 				return
 			}
+		case *core.Kv:
+			err = a.GenerateKvResources(construct, result, dag)
+			if err != nil {
+				return
+			}
```

Comment:
this needs to handle the dependency between exec unit and ddb table, or that needs to go into the execution unit file since its created second. Not sure if it actually matters for KV since we dont use env vars but my assumption is we will changge that

Reply:
nevermind this should be handled by default in generateExecUnit i believe, but this will create a table per id so that discussion i mentioned before needs to happen